src/raft.rs

```diff
@@ -1114,9 +1117,8 @@ impl<T: Storage> Raft<T> {
         self.state = StateRole::Leader;
 
         // update uncommitted state
-        let uncommitted_size = self.compute_uncommitted_size();
         let committed_idx = self.raft_log.committed;
-        self.uncommitted_state.uncommitted_size = uncommitted_size;
+        self.uncommitted_state.uncommitted_size = 0;
         self.uncommitted_state.last_committed_index = committed_idx;
```

Comment:
With `uncommitted_size` set to zore, `last_committed_index` should set to `self.raft_log.last_index()`, also the 
name `last_committed_index` may need to change